src/Settings/DeleteConfirmation.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { FunctionComponent } from "react"
+import { useTranslation } from "react-i18next"
+import { StatusBar, StyleSheet, View, ScrollView } from "react-native"
+import { showMessage } from "react-native-flash-message"
+
+import { useOnboardingContext } from "../OnboardingContext"
+import { useSymptomLogContext } from "../MyHealth/SymptomLogContext"
+
+import { Text } from "../components"
+import { TouchableOpacity } from "react-native"
+import { useStatusBarEffect } from "../navigation"
+import { SUCCESS_RESPONSE } from "../OperationResponse"
+
+import {
+  Outlines,
+  Spacing,
+  Buttons,
+  Typography,
+  Layout,
+  Colors,
+  Affordances,
+} from "../styles"
+
+const DeleteConfirmation: FunctionComponent = () => {
+  useStatusBarEffect("dark-content", Colors.secondary10)
+  const { resetOnboarding } = useOnboardingContext()
+  const { deleteAllLogEntries } = useSymptomLogContext()
+
+  const { t } = useTranslation()
+  const handleOnPressDeleteAllData = async () => {
+    const deleteLogEntriesResult = await deleteAllLogEntries()
```

Comment:
Wouldnt do this in this pr, but it might make sense to collate all of these 'delete the thing's into one spot as this view layer function might grow to be hard to follow as we add more things that need to be deleted. Having a single function to act as api would be a better separation of concerns. something like:
```typescript
  const deleteAllUserDataResult = await deleteAllUserData()
  ....
```

```typescript
  const deleteAllUserData = (): Promise<Result> => {
      return Promise.all([
           deleteThing1,
           deleteThing2,
            ....

  ....
```